src/SignalR/common/SignalR.Common/ref/Microsoft.AspNetCore.SignalR.Common.netcoreapp.cs

```diff
@@ -31,6 +31,8 @@ public partial class CloseMessage : Microsoft.AspNetCore.SignalR.Protocol.HubMes
     {
         public static readonly Microsoft.AspNetCore.SignalR.Protocol.CloseMessage Empty;
         public CloseMessage(string error) { }
+        public CloseMessage(string error, bool allowReconnect) { }
+        public bool AllowReconnect { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
```

Comment:
@aspnet/asp-net-api-reviews @davidfowl What do I need to do to get the api-approved label on this PR? I want to merge this soon since it for 3.1.